api-reference/40 Common Types/Prompt/Prompt.md

```diff
@@ -0,0 +1,12 @@
+---
+id: Prompt
+module: common/ai-integration
+export: Prompt
+type: Object
+---
+---
+##### shortDescription
+A prompt for the AI model.
+
+---
+Most AI services support a prompt that contains two fields: a system message and a user message. The system message directs the AI model, such as "You are a professional assistant." The user message is the user's request, like "Create a to-do list for today."
```

Comment:
I'd use "such as" both time. "Like" is more conversational - I don't feel that comfortable using it in technical documentation. 

```suggestion
Most AI services support a prompt that contains two fields: a system message and a user message. The system message directs the AI model, such as "You are a professional assistant." The user message is the user's request, such as "Create a to-do list for today."
```